src/ensembl/src/shared/components/external-link/ExternalLink.test.tsx

```diff
@@ -31,21 +30,24 @@ const defaultProps: ExternalLinkProps = {
 };
 
 describe('<ExternalLink />', () => {
-  const renderExternalLink = (props: Partial<ExternalLinkProps> = {}) =>
-    render(<ExternalLink {...defaultProps} {...props} />);
-
+  const renderExternalLink = () => render(<ExternalLink {...defaultProps} />);
   it('renders without error', () => {
     const { container } = renderExternalLink();
     expect(() => container).not.toThrow();
   });
 
   it('applies the passed in classNames', () => {
     const { container } = renderExternalLink();
-    expect(container.getElementsByClassName('.icon')).toBeTruthy();
+
     expect(
       container
-        .querySelector('.link')
-        ?.classList.contains(defaultProps.classNames?.link as string)
+        .querySelector(`icon-mock`)
+        ?.getAttribute('classname')
+        ?.includes(defaultProps.classNames?.icon as string)
```

Comment:
For future reference: while this totally works, it would probably be a bit more idiomatic to do this check using the element's `classList` (`container.querySelector('icon-mock')?.classList.?contains(defaultProps.classNames.icon)`).

Reply:
I tried using `classList` but doesn't seem to work for `icon-mock`.